nni/tools/nnictl/nnictl.py

```diff
@@ -243,12 +243,9 @@ def parse_args():
     def show_messsage_for_nnictl_package(args):
         print_error('nnictl package command is replaced by nnictl algo, please run nnictl algo -h to show the usage')
 
-    parser_package_subparsers = subparsers.add_parser('package', help='control nni tuner and assessor packages').add_subparsers()
-    parser_package_subparsers.add_parser('install', help='install packages').set_defaults(func=show_messsage_for_nnictl_package)
```

Comment:
Does these subcommand still supported? What is the output in `nnictl package --help` command after remove these parser here?

Reply:
all subcommand don't support, `--help` will show
```
usage: nnictl package [-h] ...

positional arguments:
  args

optional arguments:
  -h, --help  show this help message and exit
```
maybe `add_help=False` is better?

Reply:
Can you add information to explain `args` here? like `parser_package_subparsers.add_argument('args', help='explain args', nargs=argparse.PEMAINDER)`.

Reply:
fix it
```
usage: nnictl package [-h] ...

positional arguments:
  args        all subcommands have deprecated, nnictl package command is
              replaced by nnictl algo, please run nnictl algo -h to show the
              usage

optional arguments:
  -h, --help  show this help message and exit
```